01-syntax/03-data-structures/the-sum-of-ranges/sum.js

```diff
@@ -3,8 +3,34 @@
  *
  */
 /* global sum range */
-
-// → Your code here.
-
+function range(start, end, step) {
```

Comment:
Could I ask to exclude `range` from that file ?

It doesn't make sense because it's already loaded in `index.html`